doc/user_guide/checkers/features.rst

```diff
@@ -779,8 +779,8 @@ Refactoring checker Messages
 :consider-swap-variables (R1712): *Consider using tuple unpacking for swapping variables*
   You do not have to use a temporary variable in order to swap variables. Using
   "tuple unpacking" to directly swap variables makes the intention more clear.
-:use-dict-literal (R1735): *Consider using {} instead of dict()*
-  Emitted when using dict() to create an empty dictionary instead of the
+:use-dict-literal (R1735): *Consider using a dict literal '%s' instead.*
+  Emitted when dict() is used to create a dictionary instead of the
```

Comment:
This file is generated automatically when building the docs (with `doc/exts/pylint_features.py`).
The current description is not completely identical to the one in `refactoring_checker.py`, which means that every time someone builds the documentation locally the `features.rst` will show up as a modified file in `git status` afterwards.

@Pierre-Sassoulas that's a bit of a general problem also with other auto-generated files. Do you have a preference how those should be treated in a PR?

Reply:
There's https://github.com/PyCQA/pylint/pull/6736 to fix this issue, but we need to make the doc generated the same on each interpreter / operating system first (some default value need to be changed to "system's interpreter"n or "system's installed enchant's dict".

Reply:
> The current description is not completely identical to the one in `refactoring_checker.py`

True, fixed!